### ISSUE TITLE
Fix math.test_tensorflow_acosh

### ISSUE THREAD

**Issue by ivy-leaves** [unifyai/ivy#13437]
|   |   |
|---|---|
|numpy|<a href="https://github.com/unifyai/ivy/actions/runs/6374527455/job/17299266211"><img src=https://img.shields.io/badge/-success-success></a>
|jax|<a href="https://github.com/unifyai/ivy/actions/runs/6374527455/job/17299266211"><img src=https://img.shields.io/badge/-failure-red></a>
|tensorflow|<a href="https://github.com/unifyai/ivy/actions/runs/6374527455/job/17299266211"><img src=https://img.shields.io/badge/-success-success></a>
|torch|<a href="https://github.com/unifyai/ivy/actions/runs/6374527455/job/17299266211"><img src=https://img.shields.io/badge/-success-success></a>
|paddle|<a href="https://github.com/unifyai/ivy/actions/runs/6374527455/job/17299266211"><img src=https://img.shields.io/badge/-success-success></a>

**Comment by Nightcrab** [comment]
Inconsistency between Jax and other backend frameworks.

```
>>> jnp.arccosh(jnp.array(-1.-1.54074396e-33j))
Array(0.+3.1415927j, dtype=complex64, weak_type=True)
>>> tf.acosh(tf.constant(-1.-1.54074396e-33j, dtype="complex64"))
<tf.Tensor: shape=(), dtype=complex64, numpy=(3.925231e-17-3.1415927j)>
>>> torch.tensor(-1.-1.54074396e-33j)
tensor(-1.-1.5407e-33j)
>>> torch.arccosh(torch.tensor(-1.-1.54074396e-33j))
tensor(3.9252e-17-3.1416j)
```

**Comment by ivy-leaves** [comment]
Hi there,

Just to let you know, this issue has been depreciated. However, new issues related to any failing tests will be created very soon, marked distinctly with the `Failing Test` label.

Kindly stay tuned for the relevant announcements on our [Discord server](https://discord.com/channels/799879767196958751/983684937712558080) in the coming week.

Thank you for your understanding and cooperation.
